runtime/compiler/env/VMJ9Server.hpp

```diff
@@ -167,6 +167,8 @@ class TR_J9ServerVM: public TR_J9VM
    virtual uintptrj_t getFieldOffset(TR::Compilation * comp, TR::SymbolReference* classRef, TR::SymbolReference* fieldRef) override { return 0; } // safe answer
    virtual bool canDereferenceAtCompileTime(TR::SymbolReference *fieldRef,  TR::Compilation *comp) { return false; } // safe answer, might change in the future
    virtual bool instanceOfOrCheckCast(J9Class *instanceClass, J9Class* castClass) override;
+   virtual bool instanceOfOrCheckCastNoCacheUpdate(J9Class *instanceClass, J9Class* castClass) override;
+   virtual bool instanceOfOrCheckCastHelper(J9Class *instanceClass, J9Class* castClass, bool cacheUpdate);
```

Comment:
`instanceOfOrCheckCastHelper` does not have to be virtual and can be made private because nobody else outside its class needs it.

Reply:
Fixed here: 3148b1f8546548ee5ee0b0291603423f72f3f004